package.json

```
@@ -136,6 +137,7 @@
       "react/prop-types": "off",
       "react/no-children-prop": "off",
       "react/display-name": "warn",
+      "react/jsx-curly-brace-presence": "error",
```

I think there would be better to add more complex rule 

```suggestion
      "react/jsx-curly-brace-presence": ["error", { "props": "never", "children": "never" }],
```

---

Done